reviewrot/pagurestack.py

```diff
@@ -0,0 +1,137 @@
+            repo_name = user_name
+            request_url = "{}/api/0/{}/pull-requests".format(self.instance,
+                                                             repo_name)
+            log.debug('Looking for pull requests for %s -> %s',
+                      'pagure.io',  repo_name)
+        log.debug('Calling API with request_url: %s', request_url)
+        response = self._call_api(url=request_url)
+        res_ = []
+        for res in response['requests']:
+            # if namespace exists in response
+            try:
+                repo_reference = os.path.join(res['project']['namespace'],
+                                              res['project']['name'])
+            except:
+                repo_reference = res['project']['name']
+            # format pull request url
+            url = os.path.join('https://pagure.io/', repo_reference,
+                               'pull-request', str(res['id']))
+            # fetch the date pull request was filed at
+            created_date = datetime.datetime.fromtimestamp(
+                int(res['date_created'])).strftime('%Y-%m-%d %H:%M:%S')
+            # format the date pull request was filed at
+            try:
+                date = datetime.datetime.strptime(created_date,
+                                                  '%Y-%m-%d %H:%M:%S.%f')
+            except ValueError:
+                date = datetime.datetime.strptime(created_date,
+                                                  '%Y-%m-%d %H:%M:%S')
+            """ check if review request is older/newer than specified time
+            interval"""
+            result = self.check_request_state(date,
+                                              state_, value, duration)
+            if result is False:
+                log.debug("pull request '%s' is not %s than specified"
+                          " time interval", res['title'], state_)
+                continue
+            # format the time interval pull request has been filed since
+            time = self.format_duration(created_at=date)
+            res = PagureReview(user=res['user']['name'],
+                               title=res['title'],
+                               url=url,
+                               time=time,
+                               comments=len(res['comments']))
+            log.info(res)
+            res_.append(res)
+        return res_
+
+    def _call_api(self, url, method='GET'):
+        """
+        Method used to call the API.
+        It returns the raw JSON returned by the API or raises an exception
+        if something goes wrong.
+
+        Args:
+            method (str): the URL to call, can be GET, POST, DELETE, UPDATE...
+                          Defaults to GET
+
+        Returns:
+            raw JSON returned by API
+        """
+        req = self.session.request(
+            method=method,
+            url=url,
+            headers=self.header,
+            verify=not False,
```

Comment:
Any reason for `not False`? By default verify is `True` so I think you can just remove this arg.